omod/src/main/java/org/openmrs/module/ugandaemrfingerprint/fragment/controller/CheckInFragmentController.java

```diff
@@ -64,6 +66,11 @@ public SimpleObject post(@SpringBean("patientService") PatientService patientSer
             //patientQueue.setComment(visitComment);
         }
 
+        if (Context.getVisitService().getActiveVisitsByPatient(patient).isEmpty()) {
```

Comment:
@slubwama Why instantiate a controller in another controller, this seems to be a bad smell